tests/combine_ecsv_files.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python
+
+"""Simple script to combine the ecsv files from individual test runs into a single monolithic file for easier
+review.
+
+"""
+import argparse
+from astropy.io import ascii
+from astropy.table import Table, vstack
+import glob
+import os
+import sys
+import traceback
+
+# ------------------------------------------------------------------------------------------------------------
+
+
+def find_files(input_file_basepath):
+    """Find the ecsv files.
+
+    Parameters
+    ----------
+    input_file_basepath : string
+        Path to start recursive search for the .ecsv files.
+
+    Returns
+    -------
+    file_list: list
+        List of ecsv full filenames
+    """
+    # Search for ecsv files ignoring '_current' sym links to existing directories
+    file_list = glob.glob("{}**/*[!current]/*.ecsv".format(input_file_basepath), recursive=True)
+    n_found = len(file_list)
+    if n_found == 0:
+        sys.exit("No .ecsv files found. Exiting...")
+    elif n_found == 1:
+        print("{} ecsv file found.".format(n_found))
+    else:
+        print("{} ecsv files found.".format(n_found))
+    return(file_list)
+
+# ------------------------------------------------------------------------------------------------------------
+
+
+def generate_output_file(ecsv_file_list, output_filename, clobber):
+    """Generate combined output ecsv file.
+
+    Parameters
+    ----------
+    ecsv_file_list: list
+        List of ecsv full filenames
+
+    output_filename: string
+         Name of the output combined .ecsv file.
+
+    clobber : Boolean
+        Overwrite existing files with the same name as output_filename?
+
+    Returns
+    -------
+    Nothing.
+    """
+    try:
+        file_start = True
+        n_found = len(ecsv_file_list)
+        filectr = 1
+        for ecsv_filename in ecsv_file_list:
+            table_data = ascii.read(ecsv_filename, format='ecsv')  # Read ecsv file
+            padding = " "*(len(str(n_found))-len(str(filectr)))
+            if len(table_data) < 2:
+                plural_string = ""
+            else:
+                plural_string = "s"
+            print("{}{}/{}: added {} row{} from {}.".format(padding,
+                                                            filectr,
+                                                            n_found,
+                                                            len(table_data),
+                                                            plural_string,
+                                                            ecsv_filename))
+            filectr += 1
+            dataset = os.path.basename(ecsv_filename)[:-5].lower()  # scrape dataset name out of ecsv filename
```

Comment:
Using '.lower()' here seems dangerous, since Linux filenames could include upper-case dataset names and applying lower would cause the code to mis-interpret or not recognize those directory names.

Reply:
Issue resolved by commit [e207375](https://github.com/spacetelescope/drizzlepac/pull/351/commits/e20737554fd030ed8d53cc1c6ffe772ab76bf18b)